apps/publish/content/common.py

```diff
@@ -242,9 +242,12 @@ def update(self, id, updates, original):
                 # race condition.
                 published_service = get_resource_service(PUBLISHED)
                 assert published_service is not None
+
+                # TODO-ASYNC: superdesk-planning tests break because of difference between `_id` and `guid`
```

Comment:
These 2 comments aren't needed, you can see the changes in my PR here:

https://github.com/superdesk/superdesk-core/blob/32fb1ba9c9a798b01a431d153b533983675443ff/apps/publish/content/common.py#L295-L308